lib/commands/release/index.ts

```diff
@@ -0,0 +1,121 @@
+/**
+ * @license
+ * Copyright 2016-2020 Balena Ltd.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { flags } from '@oclif/command';
+import Command from '../../command';
+import * as cf from '../../utils/common-flags';
+import { getBalenaSdk, getVisuals, stripIndent } from '../../utils/lazy';
+import type * as BalenaSdk from 'balena-sdk';
+import jsyaml = require('js-yaml');
+import { tryAsInteger } from '../../utils/validation';
+
+interface FlagsDef {
+	help: void;
+	composition?: boolean;
+}
+
+interface ArgsDef {
+	commitOrId: string;
+}
+
+export default class ReleaseCmd extends Command {
+	public static description = stripIndent`
+		Get info for a release.
+`;
+	public static examples = [
+		'$ balena release a777f7345fe3d655c1c981aa642e5555',
+		'$ balena release 1234567',
+	];
+
+	public static usage = 'release <commitOrId>';
+
+	public static flags: flags.Input<FlagsDef> = {
+		help: cf.help,
+		composition: flags.boolean({
+			default: false,
+			char: 'c',
+			description: 'Return the release composition',
+		}),
+	};
+
+	public static args = [
+		{
+			name: 'commitOrId',
+			description: 'the commit or ID of the release to get information',
+			required: true,
+		},
+	];
+
+	public static authenticated = true;
+
+	public async run() {
+		const { args: params, flags: options } = this.parse<FlagsDef, ArgsDef>(
+			ReleaseCmd,
+		);
+
+		const balena = getBalenaSdk();
+		if (options.composition) {
+			await this.showComposition(params.commitOrId, balena);
+		} else {
+			await this.showReleaseInfo(params.commitOrId, balena);
+		}
+	}
+
+	async showComposition(commitOrId: string, balena: BalenaSdk.BalenaSDK) {
+		const release = await balena.models.release.get(tryAsInteger(commitOrId), {
+			$select: 'composition',
+		});
+
+		console.log(jsyaml.dump(release.composition));
+	}
+
+	async showReleaseInfo(commitOrId: string, balena: BalenaSdk.BalenaSDK) {
+		const fields: Array<keyof BalenaSdk.Release> = [
+			'id',
+			'commit',
+			'created_at',
+			'status',
+			'semver',
+			'is_final',
+			'build_log',
+			'start_timestamp',
+			'end_timestamp',
+		];
+
+		const release = await balena.models.release.get(tryAsInteger(commitOrId), {
```

Comment:
Can we convert this to a `parse` arg option?
See: https://github.com/balena-io/balena-cli/blob/e098cdca171abf9b1df2534c59f39276833b84ad/lib/commands/device/index.ts#L68


In that case make sure to change all `commitOrId` params to `string | number`.